### PR TITLE
Make torch digging instant again

### PR DESCRIPTION
**Pull request by HybridDog**
* @Fixer-007 fixes https://github.com/minetest/minetest_game/issues/1706 (torch instant mining)
* When the digging time is 0, a delay of 0.15 seconds is added between digging nodes. If the left mouse button is released, the delay is set to 0, thus quick click-digging.

### CODE REVIEW

File: src/game.cpp [hunk 1]
```diff
@@ -3824,6 +3820,16 @@ void Game::handleDigging(const PointedThing &pointed, const v3s16 &nodepos,
 			params = getDigParams(nodedef_manager->get(n).groups, tp);
 	}
 
+	if (!runData.digging) {
+		infostream << "Started digging" << std::endl;
```
Comment (nerzhul) [inline src/game.cpp]: to remove or verbose

Comment (sfan5) [inline src/game.cpp]: This is just copied from the old code

Comment (HybridDog) [inline src/game.cpp]: @nerzhul it doesn't belong to this PR

**Comment by sfan5** [comment]
What's the difference from this and just reverting the original PR?
(what did the original pr fix?)

**Comment by paramat** [comment]
This may have to reverted for release due to #5728

**Comment by HybridDog** [comment]
paramat, there's the fix: #5785
